### PR TITLE
Headings

### PR DESCRIPTION
**Pull request by SiteSeerBE**

### CODE REVIEW

File: app/src/domain/competency-management/components/root-competencies-create-page.vue [hunk 1]
```diff
@@ -45,14 +45,15 @@ const competencyFramework = computed(() =>
         <span v-t="'frameworks.route.index.heading'" />
       </UiBreadcrumbLink>
       <UiBreadcrumbLink
+        is="h1"
```
Comment (Moeriki) [inline app/src/domain/competency-management/components/root-competencies-create-page.vue:48]: Breadcrumb link is not `h1`.

**Review by Moeriki — CHANGES_REQUESTED** [review]

File: app/src/domain/competency-management/components/root-competencies-create-page.vue (hunk 1 above)
Comment (SiteSeerBE) [reply app/src/domain/competency-management/components/root-competencies-create-page.vue]: Line removed

File: app/src/ui/title/ui-subtitle.vue [hunk 2]
```diff
@@ -1,11 +1,5 @@
...
-  is?: string;
-}>();
-</script>
-
 <template>
-  <Component :is="is ?? 'div'" class="font-semibold text-gray-600 uppercase">
+  <p role="doc-subtitle" class="font-semibold text-gray-600 uppercase">
     <slot />
-  </Component>
+  </p>
```
[the remark this answers is not in the document]
Comment (SiteSeerBE) [reply app/src/ui/title/ui-subtitle.vue]: Commit dropped.

**Comment by SiteSeerBE** [comment]
Breadcrumb needed only a minor change which I pushed on this branch to avoid having to create a new one for such a small commit.

File: app/src/ui/breadcrumb/ui-breadcrumb.vue [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <template>
-  <ol>
-    <slot />
-  </ol>
+  <nav aria-label="Breadcrumb">
```
Comment (Moeriki) [inline app/src/ui/breadcrumb/ui-breadcrumb.vue:2]: Should be internationalized.

**Review by Moeriki — CHANGES_REQUESTED** [review]

File: app/src/ui/breadcrumb/ui-breadcrumb.vue (hunk 3 above)
Comment (SiteSeerBE) [reply app/src/ui/breadcrumb/ui-breadcrumb.vue]: Done

**Pull request merged**